### ISSUE TITLE
[BUG] "Tap the *star* on items" message in My Locations dining tab not accessible to screen reader users

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5759]
**Describe the bug**
"Tap the *star* on items" message in My Locations dining tab not accessible to screen reader users.
The word for star or favorites is not in English and the screen reader users may not conclude what to do. 

**To Reproduce**
_Ensure user has no Favorite dining locations selected so that message appears._ 
_Using TalkBack or VoiceOver_
1. Go to 'Favorites' section
2. Gesture to 'Dining'
3. Gesture to the 'My locations' tab

**Expected behavior**
"Tap the *star* on items" message in My Locations dining tab is accessible to screen reader users.

**Actual behavior**
"Tap the *star* on items" message in My Locations dining tab not accessible to screen reader users. the word for Favorites/star is not in audible or perhaps not in English. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/be46b293-f84b-4e0f-9330-4187b99211b9)

https://github.com/user-attachments/assets/52f085f8-a47d-4783-b756-75873569b103

**Smartphone (please complete the following information):**
- Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.51

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.51

**Additional context**
